python/packages/autogen-core/docs/src/user-guide/core-user-guide/faqs.md

```diff
@@ -34,3 +34,27 @@ worker1 = WorkerAgentRuntime(host_address=host_address, extra_grpc_config=extra_
 ```
 
 **Note**: When `WorkerAgentRuntime` creates a host connection for the clients, it uses `DEFAULT_GRPC_CONFIG` from `HostConnection` class as default set of values which will can be overriden if you pass parameters with the same name using `extra_grpc_config`.
+
+## What are model capabilities and how do I specify them?
```

Comment:
It is hard to discover this FAQ right now. Let's put this in the following places, duplicate if necessary.

1. https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/framework/model-clients.html#built-in-model-clients
2. https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/tutorial/models.html#local-models
3. https://microsoft.github.io/autogen/dev/reference/python/autogen_ext/autogen_ext.models.html#autogen_ext.models.OpenAIChatCompletionClient

Reply:
If this is specific to openai extension, should we put it in the ext docs?

Reply:
This is necessary for all model client implementations to have this. Because we are using this information to validate the model client inside agents e.g., AssistantAgent.

Reply:
1 and 2 is done, 3 already has a guide